python/src/MultigroupTable.python.cpp

```diff
@@ -36,11 +37,13 @@ void wrapMultigroupTable( python::module& module, python::module& ) {
     python::init< std::string, std::string, std::string, std::string,
                   double, double, double, double,
                   EnergyGroupStructure, FluxWeights,
-                  ReactionCrossSections >(),
+                  ReactionCrossSections,
+                  std::optional< AverageFissionEnergyRelease > >(),
     python::arg( "zaid" ), python::arg( "libname" ), python::arg( "source" ),
     python::arg( "process" ), python::arg( "awr" ), python::arg( "weight" ),
     python::arg( "temperature" ), python::arg( "dilution" ),
     python::arg( "structure" ), python::arg( "flux" ), python::arg( "xs" ),
```

Comment:
I don't like "xs" for the reaction XSs because when the total XS comes in, it might get confusing. Maybe reactionxs?

Reply:
I changed this following your remark to reaction_xs